### ISSUE TITLE
Configure memory use for the container

### ISSUE THREAD

**Issue by shelajev** [vasilmkd/docker-stats-monitor#32]
thanks for the excellent utility!
Currently neither the dockerfile nor the command line to run the container configure the memory use. Native image application will by default have the following configuration (in the environment with enough resources): 
`-Xmx` -- max heap size -- not limited by default. 
`-Xmx` -- young generation size -- 256M by default. 

the consequence of that is that currently, by default, before the young gen is filled there's no GC happening. So the application (and the docker container) will consume at least 256M on a machine where docker has enough resources. 

You can configure it on the application level by passing `-Xmx` and `-Xmn` to the native image, for example in the dockerfile 
```
ENTRYPOINT [ "/server", "-Xmx64M" ]
```

or using docker options to limit memory available to the container, for example: 
```
docker run -d --name monitor --memory=64m --memory-swap=64m -p "8080:8080" -v /var/run/docker.sock:/var/run/docker.sock vasilvasilev97/docker-stats-monitor
``` 

Then the container will work the same way, but the memory usage will stay at a fraction of 256M to which it grows now. 

Before: 
![Screenshot 2020-05-21 at 16 28 15](https://user-images.githubusercontent.com/426039/82606642-16fe1d80-9bc0-11ea-8384-55ae32dc9b61.png)

After: 
![Screenshot 2020-05-21 at 23 34 49](https://user-images.githubusercontent.com/426039/82606573-f8982200-9bbf-11ea-8c4d-2b537c93dec3.png)

**Comment by vasilmkd** [comment]
@shelajev Thank you for the awesome suggestion! I'm really new to native image and I haven't explored much of the finer details. This is invaluable knowledge. What do you suggest is the better approach? Would you like to submit a PR for the Dockerfile change and/or maybe a small section in the readme?

**Comment by vasilmkd** [comment]
Resolved in #33.
